breadcord/helpers.py

```diff
@@ -77,7 +77,7 @@ def search_for(
     threshold: float = 80,
     max_results: int | None = 25,
 ) -> Sequence[_T]:
-    """A custom implementation of a fuzzy search algorithm.
+    """Custom implementation of a fuzzy search algorithm.
```

Comment:
See previous comment @ L27.

```suggestion
    """A custom implementation of a fuzzy search algorithm.
```